utils/web_scraper.py

```
@@ -288,51 +485,277 @@ async def _fetch_weibo_trending_fallback(limit: int = 10) -> Dict[str, Any]:
+        }
+    except Exception as e:
+        logger.exception(f"获取Twitter热门失败: {e}")
+        return await _fetch_twitter_trending_fallback(limit)
+
+
+async def _fetch_twitter_trending_fallback(limit: int = 10) -> Dict[str, Any]:
+    """
+    Twitter热门的回退方案
+    使用第三方服务获取热门话题，因为Twitter官方API需要OAuth认证
+    """
+    # 第三方热门话题API列表（按优先级排序）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)